Fails to load images on old Android Versions when using ImageConfig.HARDWARE

I was using the ImageConfig.HARDWARE configuation for thumbnails since it seemed to help perfomance, but I got this exception when I was testing on a older emulator(API 24).

I did some research and it seems like `Bitmap.Config.HARDWARE` wasn't added until API 26. I changed what I use for ImageConfig to a variable that changes based on the API level, but I think this should be handled at the library level or at least give a warning.

```kotlin
2022-11-08 00:12:12.982 11097-11097/ca.gosyer.jui.android.debug W/ImageLoaderImageKt: Caused by: java.lang.NoSuchFieldError: No field HARDWARE of type Landroid/graphics/Bitmap$Config; in class Landroid/graphics/Bitmap$Config; or its superclasses (declaration of 'android.graphics.Bitmap$Config' appears in /system/framework/framework.jar)
        at com.seiko.imageloader.util.BitmapKt.toBitmapConfig(Bitmap.kt:19)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder.configureConfig(BitmapFactoryDecoder.kt:86)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder.decode(BitmapFactoryDecoder.kt:59)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder.access$decode(BitmapFactoryDecoder.kt:26)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder$decode$2$1.invoke(BitmapFactoryDecoder.kt:34)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder$decode$2$1.invoke(BitmapFactoryDecoder.kt:34)
        at kotlinx.coroutines.InterruptibleKt.runInterruptibleInExpectedContext(Interruptible.kt:51)
        at kotlinx.coroutines.InterruptibleKt.access$runInterruptibleInExpectedContext(Interruptible.kt:1)
        at kotlinx.coroutines.InterruptibleKt$runInterruptible$2.invokeSuspend(Interruptible.kt:43)
        at kotlinx.coroutines.InterruptibleKt$runInterruptible$2.invoke(Interruptible.kt)
        at kotlinx.coroutines.InterruptibleKt$runInterruptible$2.invoke(Interruptible.kt)
        at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
        at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:161)
        at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
        at kotlinx.coroutines.InterruptibleKt.runInterruptible(Interruptible.kt:42)
        at kotlinx.coroutines.InterruptibleKt.runInterruptible$default(Interruptible.kt:39)
        at com.seiko.imageloader.component.decoder.BitmapFactoryDecoder.decode(BitmapFactoryDecoder.kt:34)
        at com.seiko.imageloader.intercept.DecodeInterceptor.decode(DecodeInterceptor.kt:45)
        at com.seiko.imageloader.intercept.DecodeInterceptor.intercept(DecodeInterceptor.kt:19)
        at com.seiko.imageloader.intercept.RealInterceptorChain.proceed(RealInterceptorChain.kt:37)
        at com.seiko.imageloader.intercept.MemoryCacheInterceptor.intercept(MemoryCacheInterceptor.kt:31)
        at com.seiko.imageloader.intercept.RealInterceptorChain.proceed(RealInterceptorChain.kt:37)
        at com.seiko.imageloader.intercept.MappedInterceptor.intercept(MappedInterceptor.kt:12)
        at com.seiko.imageloader.intercept.RealInterceptorChain.proceed(RealInterceptorChain.kt:37)
        at com.seiko.imageloader.RealImageLoader.executeMain(ImageLoader.kt:64)
        at com.seiko.imageloader.RealImageLoader.access$executeMain(ImageLoader.kt:29)
        at com.seiko.imageloader.RealImageLoader$execute$2.invokeSuspend(ImageLoader.kt:50)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
        at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)
```

--- a/image-loader/src/androidMain/kotlin/com/seiko/imageloader/util/Bitmap.kt
+++ b/image-loader/src/androidMain/kotlin/com/seiko/imageloader/util/Bitmap.kt
@@ -1,9 +1,9 @@
 package com.seiko.imageloader.util
 
-import android.annotation.SuppressLint
 import android.graphics.Bitmap
 import android.os.Build
 import com.seiko.imageloader.request.Options
+import io.github.aakira.napier.Napier
 
 internal val Bitmap.safeConfig: Bitmap.Config
     get() = config ?: Bitmap.Config.ARGB_8888
@@ -11,12 +11,21 @@ internal val Bitmap.safeConfig: Bitmap.Config
 internal val Bitmap.Config.isHardware: Boolean
     get() = Build.VERSION.SDK_INT >= 26 && this == Bitmap.Config.HARDWARE
 
-@SuppressLint("NewApi")
 internal fun Options.ImageConfig.toBitmapConfig(): Bitmap.Config = when (this) {
     Options.ImageConfig.ALPHA_8 -> Bitmap.Config.ALPHA_8
     Options.ImageConfig.ARGB_8888 -> Bitmap.Config.ARGB_8888
-    Options.ImageConfig.RGBA_F16 -> Bitmap.Config.RGBA_F16
-    Options.ImageConfig.HARDWARE -> Bitmap.Config.HARDWARE
+    Options.ImageConfig.RGBA_F16 -> if (Build.VERSION.SDK_INT >= 26) {
+        Bitmap.Config.RGBA_F16
+    } else {
+        Napier.w { "ImageConfig.RGBA_F16 not support in android less than API 26" }
+        Bitmap.Config.ARGB_8888
+    }
+    Options.ImageConfig.HARDWARE -> if (Build.VERSION.SDK_INT >= 26) {
+        Bitmap.Config.HARDWARE
+    } else {
+        Napier.w { "ImageConfig.HARDWARE not support in android less than API 26" }
+        Bitmap.Config.ARGB_8888
+    }
 }
 
 internal fun Bitmap.Config?.toSoftware(): Bitmap.Config {

